### PR TITLE
Hotfix: add additional changes required for equivalence with contrib/ectrans in ifs-source

### PR DESCRIPTION
**Pull request by samhatfield**

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -61,42 +61,42 @@ foreach( prec dp sp )
         set( t 47 )
         set( grid O48 )
         ecbuild_add_test( TARGET ectrans_test_benchmark_${prec}_T${t}_${grid}_mpi${mpi}_omp${omp}_nfld0
-            COMMAND ectrans-benchmark-${prec} ARGS --truncation ${t} --grid ${grid} --niter 2 --nfld 0 -v --meminfo --check 100
+            COMMAND ectrans-benchmark-${prec} ARGS --truncation ${t} --grid ${grid} --niter 2 --nfld 0 -v --meminfo --check 100 --norms -v
```
Comment (wdeconinck) [inline tests/CMakeLists.txt:64]: duplicate -v argument

**Review by wdeconinck — APPROVED** [review]
I prefer to keep the file to be named `ectrans_compile_options.cmake` if you don't mind.

File: tests/CMakeLists.txt (hunk 1 above)
Comment (wdeconinck) [reply tests/CMakeLists.txt]: Fixed it.

**Comment by samhatfield** [comment]
Sure thing, happy to keep the old name.

**Pull request merged**